### PR TITLE
Fix embeddings to support local file

### PR DESCRIPTION
**Pull request by yuwenzho**
## Type of Change

bug fix
API changed or not: no

## Description

Fix embeddings to support local file

## How has this PR been tested?

CI

## Dependency Change?

No

### CODE REVIEW

File: intel_extension_for_transformers/langchain/embeddings/optimized_sentence_transformers.py [hunk 1]
```diff
@@ -53,12 +54,84 @@ def __init__(self, *args, **kwargs):
...
+                                    self._model_config['__version__']['sentence_transformers'],
+                                    sentence_transformers.__version__))
+
+        # Check if a readme exists
+        model_card_path = sentence_transformers.util.load_file_path(
+            model_name_or_path, 'README.md', token=token, cache_folder=cache_folder)
+        if model_card_path is not None:
+            try:
+                with open(model_card_path, encoding='utf8') as fIn:
+                    self._model_card_text = fIn.read()
```
Comment (kevinintel) [inline intel_extension_for_transformers/langchain/embeddings/optimized_sentence_transformers.py:98]: Do you need _model_card_text?

Comment (yuwenzho) [reply intel_extension_for_transformers/langchain/embeddings/optimized_sentence_transformers.py:98]: The logic for handling README file exists in the parent class's original [implementation](https://github.com/UKPLab/sentence-transformers/blob/master/sentence_transformers/SentenceTransformer.py#L831-L838).
Even if the README file does not exist, the raised exception will be passed without causing any issues.

**Comment by XuhuiRen** [comment]
recommend to offline download a embedding model before running the CI test

**Comment by VincyZhang** [comment]
> recommend to offline download a embedding model before running the CI test

This is what we do now

**Review by XuhuiRen — APPROVED** [review]

**Pull request merged**